frontend/src/scenes/project-homepage/ProjectHomepage.tsx

```diff
@@ -43,16 +42,7 @@ export function ProjectHomepage(): JSX.Element {
             >
                 Invite members
             </LemonButton>
-            <LemonButton
-                data-attr="project-home-new-insight"
```

Comment:
Hey @gitstart , this looks great!

The only change I'd make is to pass through the `data-attr` prop. It should remain `project-home-new-insight` for the button on the project homepage, and `saved-insights-new-insight-button` on the saved insights page.